lua/compe_snippets_nvim/init.lua

```diff
@@ -1,5 +1,11 @@
 local compe = require("compe")
 local Source = {}
+local ok, snippets_nvim = pcall(require, "snippets")
+
+if not ok then
+  error("You need to install snippets.nvim!")
```

Comment:
Is this implementation correct?

Reply:
Maybe good.
To be honest, I don't familiar to Lua...😅

Reply:
This gives an error when `snippets.nvim` isn't installed, even when I explicitly set `compe.setup.source.snippets_nvim = false`

Reply:
Thank you for your report.
I will fix it soon.

Reply:
I've pushed patch for it.
Could you test with latest master?

Reply:
@hrsh7th Yes, it works now! Awesome work, I really enjoy using nvim-compe!